docs/pages/choose-an-edition/teleport-cloud/faq.mdx

```diff
@@ -138,40 +138,46 @@ The ability to download recordings for offline viewing will be available in a fu
 
 ### Will Teleport be automatically upgraded with each release?
 
-Teleport Cloud follows the Teleport [Upcoming
+Teleport Team and Teleport Enterprise Cloud follow the Teleport [Upcoming
 Releases](../../upcoming-releases.mdx) timelines.
 
-- Major version upgrades for customer Auth and Proxy Servers occur after the
-  first minor release (for example (=teleport.major_version=).1.0) of Teleport.
+- Major version upgrades for cloud Teleport Auth Service and Teleport Proxy Service occur 
+  after the first minor release (for example (=teleport.major_version=).1.0) of Teleport.
   This typically happens within one month of a new major release of Teleport
   becoming available.
 - After the first minor release, subsequent minor and patch upgrades typically
   occur within a week.
 
 <Admonition type="note" title="Minimum agent version">
-    For major version upgrades (e.g. 12.x to 13.x), your Teleport tenant will NOT be automatically upgraded if any
-    Teleport agents connected to the cluster are not compatible (> 1 major version behind).  This is done to ensure
-    backwards compatibility. To get version information on your agents versions
-    see [here](#how-can-i-find-version-information-on-connected-agents).
+    For major version upgrades (for example, 12.x to 13.x), your Teleport tenant is NOT 
```

Comment:
```suggestion
    For major version upgrades (for example, 12.x to 13.x), your Teleport account is NOT 
```
I think we were moving from "tenant" to "account"

Reply:
Calling it your "account" seems a little far removed from cluster/instance/tenant to me, but if that's the direction we're headed, that's fine.